templates/zerver/help/import-from-slack.md

```diff
@@ -0,0 +1,50 @@
+# Import from Slack
+
+To import users and channels from Slack to Zulip please follow
+bellow steps:
+
+1. Generate Slack API Token from [here](https://api.slack.com/docs/oauth-test-tokens).
+   Test tokens are enough to import all of needed data.
+
+2. Go to **Settings** by clicking on cog from top right corner.
```

Comment:
Replace step 2 with the `{!settings.md!}` macro documented [here](http://zulip.readthedocs.io/en/latest/user-docs.html#settings-settings-md-macro)